src/plugins/auto/executable_network.cpp

```diff
@@ -90,6 +90,7 @@ MultiDeviceExecutableNetwork::MultiDeviceExecutableNetwork(const DeviceMap<Infer
         auto& network = networkValue.second;
         GenerateWorkers(device, network);
     }
+    _requestWrapper = std::make_shared<AutoRequestWrapper>();
```

Comment:
@wangleis  currently MULTI logic is not changed in this PR, do we need to apply this change for MULTI too in this PR, like MULTI:CPU, MULTI:GPU, return device inferrequest directly

Reply:
@tiger100256-hu I would prefer to separate changes for AUTO and MULTI. Let's review this PR for AUTO with Maxim first. We can create another PR for MULTI if it is needed.